src/app/containers/CpsTable/CpsTableCell/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { arrayOf, string } from 'prop-types';
+
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { textBlockPropTypes } from '#models/propTypes/text';
+
+import Blocks from '../../Blocks';
+import CpsText from '../../CpsText';
+
+const StyledTd = styled.td`
+  border-bottom: 1px solid #dbdbdb;
```

Comment:
These colours come from the existing Sport colours here - https://github.com/bbc/gs-sass-tools/blob/master/settings/_sport-colours.scss#L14
Should I open a PR to add these two colours to psammead?

Reply:
yeah it's a bit annoying but that's our source of truth for colours, just makes clear what UX etc refer to them as and allows us to change them en masse e.g. for chameleon project.

Feel free to do this in a follow-up PR though

Reply:
Sorry was looking into this earlier and got distracted - I did some comparison and it seems the colour here is different to what is on the sport pages like https://www.bbc.co.uk/sport/cricket/54646061

That page seems to have a border-bottom of `#eee` rather than `#dbdbdb` - the hover background colour seems to be slightly different too

Not sure if it's an issue.  Either way, none of these exact colours were on our pallette.  If we wanted to use the variables we'd need to either add new ones to psammead, or update the components to use one of the existing variables.

Happy to help with either of these :)

Reply:
Ah good spot! I actually copied these colours from AlephAMP rather than Morph - https://www.bbc.co.uk/sport/amp/cricket/54646061

I'll open a follow up PR to add these to psammead :)